VirtoCommerce.Storefront/Domain/Cart/CartBuilder.cs

```diff
@@ -449,18 +450,20 @@ public virtual Task RemoveShipmentAsync(string shipmentId)
 
         #endregion      
 
-        protected virtual cartModel.ShoppingCartSearchCriteria CreateCartSearchCriteria(string cartName, Store store, User user, Language language, Currency currency)
+        protected virtual WishlistSearchCriteria CreateCartSearchCriteria(string cartName, Store store, User user, Language language, Currency currency, string type)
```

Comment:
CardBuilder hasn't to know anything about Wishlists, need to change back to ShoppingCartSearchCriteria

Reply:
fixed